nlp_architect/data/utils.py

```diff
@@ -154,3 +148,39 @@ def sample_label_unlabeled(samples: List[InputExample], no_labeled: int, no_unla
     label_samples = [samples[i] for i in labeled_indices]
     unlabel_samples = [samples[i] for i in unlabeled_indices]
     return label_samples, unlabel_samples
+
+
+def split_column_dataset(
+        m: int, n: int, out_folder, dataset, m_filename='m.txt', n_filename='n.txt', tag_col=-1):
```

Comment:
* do not specify output file names, let the user choose
* please rename m/n to a more semantic name